### ISSUE TITLE
Asura Wrath lacks any blur effects after #8984

### ISSUE THREAD

**Issue by frankhval** [RPCS3/rpcs3#9475]
## Quick summary

I've been playing the game(Asura's Wrath, [BLUS30721] ) mostly fine using the build from 2019(rpcs3-v0.0.6-7909-a254a203_linux64 ) but after I've downloaded the recent build for linux( rpcs3-v0.0.13-11444-e449111c_linux64 ) any blur effects(both motion blur and a depth-of-field blur) ceased to exist.

here's the screenshot from 0.0.6:
![Screenshot from 2020-12-21 06-10-14](https://user-images.githubusercontent.com/76416457/102735502-b3092680-4353-11eb-9e24-655209169766.png)

and here is from 0.0.13:
![Screenshot from 2020-12-21 06-12-28](https://user-images.githubusercontent.com/76416457/102735512-b7354400-4353-11eb-9532-fa201c495cb7.png)

I've launched builds back to back on a same machine with a same config file etc.

Notice a lack of any depth-of-field effects on ships on a newer build's screenshot. Motion blur post effects are not present in a new build as well.

After a hint I've found the commit that broke posteffects:  #8984

#### 6. Please provide your system configuration:
* OS : Ubuntu 18.04
* CPU : i7-6700HQ
* GPU : eForce GTX 965M
* Driver version : 450.80.02

I'm not sure but I do think the issue is more with unreal engine 3 and fixing it would fix similar issues with other games using the engine as well.

**Comment by Joaozin-tech** [comment]
updates drive GPU and build rpsc3 retest friend

**Comment by frankhval** [comment]
> updates drive GPU and build rpsc3Linux retest friend

Sadly there are no newer version of drivers yet, I'm using the latest stable one from september 2020 and it is kinda given that programs should work with a stable driver well, erm, stable. 

And anyway anything more complicated than "make install" is way above my league, therefore I'm using appimages which are kinda self-contained as I understand the thing.

But you gave me an idea to test out builds that were made before the driver came out(30.09.2020) and I've found the commit that broke post effects in the game:

I've tested builds from september to october and here are the results:

2020-09-16: 0.0.12-10888: working
2020-09-25: 0.0.12-10942: working
2020-09-27: 0.0.12-10950: working
2020-09-27: 0.0.12-10953: not working
2020-09-27: 0.0.12-10954: not working
2020-09-28: 0.0.12-10957: not working
2020-10-01: 0.0.12-10967: not working
2020-10-10: 0.0.12-10989: not working

I suppose, if I'm reading it right, the commit that broke posteffects was in a build 10953, which is #8984 and it kinda make sense since there was a call for testers because the commit broke something in other games too.

I'll update the description accordingly.

**Comment by frankhval** [comment]
I've done a little more testing because some of the other issues mentioning #8984 were blaming broken things on nvidia vulkan drivers so I've come forward and test the most recent build with openGL instead of vulkan and yes, with openGL everything is working as expected(and I would say with considerably more stable fps than with vulkan).

I'm not convinced it's something that exclusively nvidia drivers are to blame though because before the pull request everything worked just fine on vulkan as well so it's still a regression after all.

**Comment by AniLeo** [comment]
No, if it's that PR and that issue with these drivers I can almost certainly assert it's the NVIDIA driver bug, but I'll wait for you to test instead of closing right away.

Latest Linux NVIDIA drivers are not from September, they're 460.89 from last week. If your distro has outdated packages you will need to find a way to install up-to-date drivers, if it's Ubuntu you can probably do it via PPA, or just consider switching to a distro that doesn't have awfully outdated repositories.

**Comment by frankhval** [comment]
I've tested on new stable nvidia drivers 460.x.x and can confirm that the issue is solved. (I've got no sound with the game with newer than 0.12.x builds though but this is a separate issue I suppose)
